misc_functions.lua

```diff
@@ -157,7 +157,7 @@ s_protect.load_config = function()
 	io.close(file)
 
 	-- Load existing config
-	simple_protection = {}
+	local simple_protection = {}
```

Comment:
This is intentional to support `simple_protection` names within the Lua config file.

Reply:
Then shouldn't it be declared outside of a function?

Reply:
It's a temporary global variable. I know, it's not nice and could perhaps be replaced by `rawset` to avoid the warning.